afs-cassandra/src/main/java/com/powsybl/afs/cassandra/CassandraUtil.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.cassandra;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
+
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.selectFrom;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public final class CassandraUtil {
+
+    private CassandraUtil() {
+    }
+
+    public static void print(CassandraContext context, PrintStream out) {
```

Comment:
well these method add print, I dont know if it is usefull but a Logger would be better

Reply:
I'm not even sure these methods are used, I'll have to check later